### ISSUE TITLE
Vertice orientation - Ask for help

### ISSUE THREAD

**Issue by pasoneto** [vchoutas/smplx#183]
When I call  `model = smplx_model.forward(transl=trans, **poses)`
I have access to vertice locations by:

`vertices = bodyModel.vertices.detach().numpy()
`

If I'm not mistaken, these vertices are represented as 3D coordinates in an Euclidian plane. However, I would like to get their orientation, be it in axis angles or quaternions. Would that be possible to do? In which case, would the orientation be in relation to the closest joint? or to the root of the kinematic tree?

Thanks in advance for any help.
